sdk/core/azure-core-tracing-opentelemetry/src/test/java/com/azure/core/tracing/opentelemetry/OpenTelemetryHttpPolicyTests.java

```diff
@@ -112,7 +112,7 @@ public void openTelemetryHttpPolicyTest() {
         request.setHeader(HttpHeaderName.USER_AGENT, "user-agent");
 
         try (Scope scope = parentSpan.makeCurrent()) {
-            createHttpPipeline(azTracer).send(request, tracingContext).block().close();
+            createHttpPipeline(azTracer).send(request, tracingContext).block().getBodyAsBinaryData();
```

Comment:
I'm curious why the change from close to getBodyAsBinaryData?

Reply:
to test that we end span when we end reading the body and not just when the response is closed  - apparently we're not great at closing response, but very good at reading body to the end on storage.